client/src/components/organisms/CreateTemplateModal.tsx

```diff
@@ -28,7 +28,7 @@ export const CreateTemplateModal: React.FC = () => {
     setAgendaNewChoice(e.target.value);
   };
   const onNewChoiceSubmit = () => {
-    if (!agendaChoice.includes(agendaNewChoice)) {
+    if (agendaNewChoice.length > 0 && !agendaChoice.includes(agendaNewChoice)) {
```

Comment:
```suggestion
    if (!!agendaNewChoice && !agendaChoice.includes(agendaNewChoice)) {
```
갠취긴 한데 이렇게 작성하면 조금 더 깔끔해용


```suggestion
    if (agendaNewChoice && !agendaChoice.includes(agendaNewChoice)) {
```
`""`는 falsy한 값이라서 요렇게 해도 작동하는데 쪼금 implicit한 면이 있긴 하죠